trial/src/main/kotlin/com/hartwig/actin/trial/TrialIngestion.kt

```diff
@@ -22,9 +30,11 @@ data class UnmappableCohort(val cohortId: String, val mappingErrors: List<Eligib
 
 data class EligibilityMappingError(val inclusionRule: String, val error: String)
 
+data class TrialWrapper(val trial: Trial, val rules: List<EligibilityRule>)
```

Comment:
Often when adding these named tuples we'll name them something like `TrialAndEligibilityRules` for clarity.